config.nims

```diff
@@ -0,0 +1,9 @@
+# begin Nimble config (version 2)
+
+# allow imports relative to the src/playdate folder from anywhere
+# ie. import domain/lcdbitmap
+--path:"./src/playdate"
```

Comment:
Not sure if this stil the place where this should be put after the build binary refactor

Reply:
In addition, this doesn't seem to resolve when the working directory is a game outside of this project (Wheelsprung)

Not sure how to do this properly; actually. I don't care too strongly, I could revert this commit so that relative "../bindings/graphics" are used again